core/dbt/parser/schemas.py

```diff
@@ -257,6 +258,7 @@ def build_test_node(self, test_target, package_name, test, root_dir, path,
 
         source_package = self.all_projects.get(package_name)
         if source_package is None:
+            # TODO : model_name is not defined here...
```

Comment:
I think s/model_name/test_target fixes this? I'd have to check the types to be 100% on that.

Reply:
when i run my pep8 linter, it (surprisingly frequently) shows that vars aren't defined. I'll make this quick fix here, but wanted to flag it in the initial commit so i didn't forget about it

Reply:
Are you using `pylint` instead of `pep8` itself to get those? It's great for this stuff but dbt makes it cry so much I can't stand using it.

Reply:
think i'm using [flake8](http://flake8.pycqa.org/en/latest/). Can try to dig up where my configs are defined -- pretty sure I'm not using the stock settings but can't seem to find them right now

Reply:
Cool, I think that's enough to go on. maybe we should add it to our CI process instead of `pep8` itself? I'm pretty sure `flake8` is just `pep8` + a mccabe complexity thing + `pyflakes`

Reply:
Ok. I'll open an issue about moving it - it seems like something we should do unless we find an unbearable false positive rate. In the meantime I've installed it in my editor so I'll fix stuff as I work on other things.